client/src/modules/Tasks/components/TaskModal/TaskModal.tsx

```diff
@@ -47,36 +52,60 @@ export function TaskModal({
       ),
     [tasks],
   );
+  const taskTypes = useMemo(
```

Comment:
you don't need it inside component as it's static

Reply:
thanks, fixed.